self-run-agents/ami-build/scripts/set-instance-protection.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Check Pre-Reqs, and that we're running on an AWS Instance Seemingly.
+# This script exist for compatibility but it is a no-op
+exit 0
```

Comment:
Should we perhaps `exit 1;` so errors are run when this tries to get run but somehow didn't get overwritten

Reply:
We don't have continue on failure in https://github.com/envoyproxy/envoy/blob/master/.azure-pipelines/bazel.yml#L20, so that would break all CI jobs meanwhile.